dockerfiles/alpine_3.15

```diff
@@ -14,7 +14,7 @@ ENV TARANTOOL_VERSION=${TNT_VER} \
     TARANTOOL_DOWNLOAD_URL=https://github.com/tarantool/tarantool.git \
     TARANTOOL_INSTALL_LUADIR=/usr/local/share/tarantool \
     GPERFTOOLS_REPO=https://github.com/gperftools/gperftools.git \
-    GPERFTOOLS_TAG=gperftools-2.9.1 \
+    GPERFTOOLS_TAG=gperftools-2.10 \
     LUAROCKS_URL=https://github.com/tarantool/luarocks/archive/6e6fe62d9409fe2103c0fd091cccb3da0451faf5.tar.gz \
```

Comment:
Why do we use a commit from year 2017? https://github.com/tarantool/luarocks/commit/6e6fe62d9409fe2103c0fd091cccb3da0451faf5

Reply:
I guess it was moved without any changes from alpine_3.9, that's all.